code/game/objects/items.dm

```diff
@@ -530,3 +539,14 @@ var/global/image/fire_overlay = image("icon" = 'icons/goonstation/effects/fire.d
 
 /obj/item/proc/is_equivalent(obj/item/I)
 	return I == src
+
+/obj/item/Crossed(atom/movable/AM)
+	if(ishuman(AM))
+		var/mob/living/carbon/human/H = AM
+		if(prob(trip_bonus))

```

Comment:
You could probably move this to be the first thing checked for, since a `prob` check is likely cheaper than an `ishuman` check.

No sense in running an `ishuman` check for something every time it's crossed unless it *needs* to run.